components/meta.tsx

```diff
@@ -15,9 +15,30 @@ export function DocMeta(
   const description = cleanMarkdown(doc);
   const href = item ? `${url}${url.endsWith("/") ? "" : "/"}~/${item}` : url;
   const imageUrl = new URL(`/img/${href}`, base).toString();
-  const title = item
-    ? `Deno Doc - ${getUrlLabel(url)} - ${item}`
-    : `Deno Doc - ${getUrlLabel(url)}`;
+  const parsed = parseURL(url);
+  let title;
```

Comment:
Nitpick: might be good to extract this out to a function (`const title = getTitle(url);`). That would also make it more easily unit testable in the future.

Reply:
yeah, I had a half a mind to do it, but there was a few things going on here that I wanted to do a bigger refactor later...